Client/Frontend/Home/ActivityStreamTopSitesCell.swift

```diff
@@ -137,6 +138,7 @@ class TopSiteItemCell: UICollectionViewCell {
 
     func configureWithTopSiteItem(site: TopSiteItem) {
         titleLabel.text = site.urlTitle
+        accessibilityValue = site.urlTitle
```

Comment:
I add the accessibilityValue. This is the user-facing string that indicates an individual top site icon.